docs/webhooks/Secrets.md

```diff
@@ -4,24 +4,54 @@ We sign our webhooks using a secret, to prove that it is WealthKernel sending yo
 
 The value is the same for all subscriptions and event types, and consists of a series of cryptographically strong bytes encoded into base64. It is not possible to receive webhooks without first generating a secret.
```

Comment:
I find it clearer to have `The secret` instead of `The value`.  I didn't realise it was referencing the secret value at first sight